### ISSUE TITLE
cordova-plugin-zeroconf - 'watch' not working [ios]

### ISSUE THREAD

**Issue by Nudelsieb** [ionic-team/capacitor#734]
```javascript
if (window.cordova) {
    const zeroconf = window.cordova.plugins.zeroconf;
    zeroconf.watch('_http._tcp.', 'local.', result => {
      var action = result.action;
      var host = result.service;
      if (action == 'added') {
        console.log('service added', host);
      } else if (action == 'resolved') {
        console.log('service resolved', host);
      } else {
        console.log('host removed', host);
      }
    });
  }
```

Xcode reports as follows.
> To Native Cordova ->  ZeroConf watch ZeroConf336797565 ["options": [_http._tcp., local., any]]
Error: Plugin Optional("ZeroConf") does not respond to method call watch:.
Ensure plugin method exists and uses @objc in its declaration

When I put a debugger in I see the method exits. 
```
watch: {
   function(type, domain, success, failure)
   arguments: TypeError: 'arguments', 'callee', and 'caller' cannot be accessed in this context.
   caller: TypeError: 'arguments', 'callee', and 'caller' cannot be accessed in this context.
   length: 4
   name: "watch"
   prototype: watch {}
   Function Prototype
}
```

**Comment by Nudelsieb** [comment]
I fixed it with the help of #612 .

`capacitor.config.json`:
```json
"ios": {
    "cordovaSwiftVersion": "3.2"
}
```

**Comment by jcesarmobile** [comment]
Nice, we forgot to document that preference, just sent a PR documenting it. This issue will be closed once it's merged.
